packages/rum/src/domain/record/serialize.ts

```diff
@@ -401,13 +416,26 @@ function getAttributesForPrivacyLevel(
   /**
    * Serialize the scroll state for each element only for full snapshot
    */
-  if (serializationContext === 'full-snapshot') {
-    if (element.scrollLeft) {
-      safeAttrs.rr_scrollLeft = Math.round(element.scrollLeft)
-    }
-    if (element.scrollTop) {
-      safeAttrs.rr_scrollTop = Math.round(element.scrollTop)
-    }
+  let scrollPositions
+  switch (serializationContext) {
+    case SerializationContext.INITIAL_FULL_SNAPSHOT:
+      scrollPositions = {
+        scrollTop: element.scrollTop && Math.round(element.scrollTop),
+        scrollLeft: element.scrollLeft && Math.round(element.scrollLeft),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏is testing for scroll position truthiness useful here?
```suggestion
        scrollTop: Math.round(element.scrollTop),
        scrollLeft: Math.round(element.scrollLeft),
```